cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -311,7 +311,8 @@ func NewControllerInitializers() map[string]InitFunc {
 	controllers["disruption"] = startDisruptionController
 	controllers["statefuleset"] = startStatefulSetController
 	controllers["cronjob"] = startCronJobController
-	controllers["certificatesigningrequests"] = startCSRController
+	controllers["certificatesigningrequestssigning"] = startCSRSigningController
+	controllers["certificatesigningrequestsapproving"] = startCSRApprovingController
```

Comment:
Changing the controller name needs a release note since people can address it via `--controllers`. If we're changing it, shorter might be better? Something like `csrsigner` / `csrautoapprover`?